app/backend/wells/views.py

```diff
@@ -256,6 +259,23 @@ class WellTagSearchAPIView(ListAPIView):
     )
 
 
+class WellSubmissionsListAPIView(ListAPIView):
```

Comment:
Many ways to skin a cat - but you could also leverage of a more generic approach (see https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering)